olp-cpp-sdk-core/include/olp/core/client/BackdownStrategy.h

```diff
@@ -19,22 +19,22 @@
 
 #pragma once
 
+#include <olp/core/CoreApi.h>
```

Comment:
Please un-sort and add an empty new line between `<random>` and `<olp/core/CoreApi.h>`

Reply:
Sorry it changed automatically when I saved the file.
Corrected and added an empty line.